lms/services/hubspot/_client.py

```diff
@@ -18,6 +35,66 @@ def get_companies(self):
         ]
         yield from self._get_objects(self._api_client.crm.companies, fields)
 
+    def create_company_import(
+        self, contract_year_billables: list[tuple[str, int, int]]
+    ):
+        """Start an import data from a CSV file to Hubspot to update companies.
```

Comment:
I think "Create an import" or "Start an import" might not be quite the right wording here, I think the method actually _does_ an import of the given `contract_year_billables` into HubSpot.

Also it's not importing a CSV file, it's importing the given `contract_year_billables` list, the fact that it has to write `contract_year_billables` to a temporary CSV file on disk is an implementation detail not part of the method's contract.

Maybe something like this?

```suggestion
    def import_billables(
        self, billables: list[tuple[str, int, int]]
    ):
        """Import the given billables into HubSpot.
```